devito/arch/compiler.py

```diff
@@ -400,6 +400,17 @@ def __lookup_cmds__(self):
         self.MPICXX = 'mpicxx'
 
 
+class ArmCompiler(GNUCompiler):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        platform = kwargs.pop('platform', configuration['platform'])
+        # Graviton flag
+        if platform is GRAVITON:
+            self.cflags += ['-mcpu=neoverse-n1']
```

Comment:
really =native doesn't do the same?

Reply:
According to the aws this flag should be used yeah